applications/app/services/InteractivePicker.scala

```diff
@@ -28,10 +29,10 @@ object InteractivePicker {
     if (!str.startsWith("/")) ("/" + str) else str
   }
 
-  def dateIsPostTransition(date: String): Boolean = {
+  def dateIsPostTransition(date: DateTime): Boolean = {
```

Comment:
Worth removing the comment below this now as redundant. Using an actual date type here is so much better :) than before.